extensions/integration-tests/src/utils.ts

```diff
@@ -0,0 +1,48 @@
+import assert = require('assert');
+import * as sqlops from 'sqlops';
+import * as vscode from 'vscode';
+import { TestServerProfile } from './testConfig';
+
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export async function waitForCompletion(thenable: Thenable<any>): Promise<any> {
```

Comment:
is this function necessary? Based on the implementation it should just be the same as awaiting on the `thenable` directly

Reply:
If it is necessary I'd suggest parameterizing it `export async function waitForCompletion<T>(thenable: Thenable<T>): Promise<T>` so that you don't need type assertions everywhere you call it

Reply:
👍 I didn't know you can await the Thenable, LOL. just tried it and this is not necessary.